internal/outofband/bmc.go

```diff
@@ -71,15 +82,21 @@ func (b *bmc) Open(ctx context.Context) error {
 }
 
 // Close logs out of the BMC
-func (b *bmc) Close() error {
+func (b *bmc) Close(ctx context.Context) error {
```

Comment:
You might consider calling the parameter `traceCtx` or something to avoid the confusion you discuss in the comments below.